test/validation/api/ipsec/ipsec_test_out.c

```diff
@@ -45,16 +45,17 @@ static void test_out_ipv4_ah_sha256(void)
 
 static void test_out_ipv4_ah_sha256_tun_ipv4(void)
 {
-	uint32_t src = IPV4ADDR(10, 0, 111, 2);
-	uint32_t dst = IPV4ADDR(10, 0, 222, 2);
-	odp_ipsec_tunnel_param_t tunnel = {
```

Comment:
I'm curious - what was the compiler error message here?

Reply:
Missing initialization on some fields (-Wmissing-field-initializers). The warning was disabled and I enable it again in this patch set, which causes an older version of GCC in CentOS 7 to fail here.